open_academy/models/course.py

```diff
@@ -0,0 +1,23 @@
+from odoo import models, fields
+
+
+class Course(models.Model):
+    _name = "openacademy.course"
+    _description = "Courses"
+
+    title = fields.Char(string="Courses", required=True)
+    description = fields.Text()
+
+    responsible_user = fields.Many2one(comodel_name="res.users", ondelete="set null")
+
+
+class Session(models.Model):
+    _name = "openacademy.sessions"
```

Comment:
Use name in singular form please https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models

```suggestion
    _name = "openacademy.session"
```